docs/tutorials/openvino-mlflow.md

```diff
@@ -0,0 +1,611 @@
+    2021/11/16 19:06:47 Pushing the code to the git repository...
+    Codeset http://gitea.34.118.93.249.nip.io/cifar10/keras.git successfully registered
+    Saving new username into config file as current username.
+    ```
+
+    The codeset and its contents can be explored in the Gitea dashboard by following the printed URL, as shown below.
+
+
+    <figure markdown> 
+      ![Gitea Codeset](img/openvino-mlflow/gitea.png)
+      <figcaption>Codeset Details in the Gitea Dashboard</figcaption>
+    </figure>
+
+    The Keras codeset includes a `MLproject` file. This is an MLFlow convention used to specify software dependencies, entrypoints and input parameters for the code present in the codeset. This vital piece of information is leveraged by the FuseML workflow that will be configured at the next step to determine automatically which software packages it needs to install and how to run the code in the codeset to train the Keras model.
+
+2. Create a FuseML workflow:
+
+    Before configuring the FuseML workflow used in this tutorial, take some time to explore it by looking at its definition in the `examples/workflows/mlflow-multi-e2e.yaml` file. Observe the following:
+    
+    - the workflow has 3 different predictor steps, each one targeting a different serving solution (OVMS, KServe and Seldon Core). They all take in the same model artifact that comes out of the training step.
+    - the OVMS predictor needs the model to be converted to the Intel IR (Intermediate Representation) format. This is facilitated by a converter step.
+    - there are explicit CPU resource requests specified for the training and OVMS serving steps. They are meant to speed up the training process and to give the OVMS predictor access to an entire vCPU. Feel free to edit the workflow and change the resource requirements to fit your scenario. Note that the Kubernetes cluster needs to have these resources available, otherwise the workflow will fail because the step containers cannot be scheduled.
+    - the default values for the workflow input parameters specify training the model for 30 epochs. This will take approximately 40 minutes and yield an accuracy close to 0.8. You can modify these parameters to experiment with different values.
+
+    To configure the FuseML workflow:
+
+    ```bash
+    $ fuseml workflow create examples/workflows/mlflow-multi-e2e.yaml
+    Workflow "mlflow-multi-e2e" successfully created
+    ```
+
+
+3. Assign the `mlflow-multi-e2e` workflow to the `keras` codeset:
+
+    ```bash
+    $ fuseml workflow assign -c keras -p cifar10 -n mlflow-multi-e2e
+    Workflow "mlflow-multi-e2e" assigned to codeset "cifar10/keras"
+    ```
+
+4. Wait for the workflow run to finish running:
+
+    By assigning the workflow to the codeset, a workflow run will be created. You can check the status
+    of the workflow run by running the following command:
+
+    ```bash
+    fuseml workflow list-runs
```

Comment:
I noticed that for some commands you include a `$` before the call, while others don't. I think it would be good have all command calls on the same way.

Reply:
done